scripts/importPublicKey.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+echo "${PUBLIC_KEY}" >> ~/.ssh/authorized_keys
+passwd -d root
+chown root:root /root/.ssh/authorized_keys
+
+apt-get update
+apt-get install -y python
```

Comment:
Why not installing this packages all together instead one-by-one?

Reply:
because if one fails all fail and if you are using ubuntu 16.04 gpg is not even a package and in 18.04 its needed

Reply:
Yeah, but they are all needed to run the system or not? Then it is ok if it fails if 1 package fails!

And if there is a difference between 16.04 and 18.04 there should be defined different packages for the system for example through a specific file which will be included based on the detected system

Reply:
Im not gonna implement an OS detecting mechanism for an development environment so just install python everything else has the role to install anyway